crates/matrix-sdk-ui/src/timeline/controller/observable_items.rs

```diff
@@ -169,7 +169,7 @@ pub struct ObservableItemsTransaction<'observable_items> {
 }
 
 impl<'observable_items> ObservableItemsTransaction<'observable_items> {
-    /// Get a referene to the timeline index at position `timeline_item_index`.
+    /// Get a reference to the timeline index at position `timeline_item_index`.
```

Comment:
Did `typos` notice that? If not, would be good to add it here: https://github.com/crate-ci/typos/issues/1156.

Reply:
Fixed, and it's been detected by `typos` already.